help/submit_pdf.md

```diff
@@ -54,6 +54,24 @@ one of these free conversion utilities:
 (Please contact us if this information is out of date or if you can
 suggest other free tools that are better.)
 
+<span id="readable"></span>PDF must be machine readable 
+-------------------------------------------------------
+
+arXiv relies on our submitters to provide PDF which are **machine readable**
+i.e. that the PDF is not encrypted, bitmapped, or otherwise altered to prevent
+text extraction. Providing a machine readable PDF facilitates *discovery and access*. 
+This means that we can build tools around your work to provide
+content discovery (e.g. full text search), and that potential readers can still
+understand its contents using tools such as screen readers. 
+
+arXiv does not accept bitmapped or scanned PDF, even in the case of software provided OCR
+(optical character recognition). Why? Partly because of the above reasons regarding 
+discovery and access, and partly because OCR software may not yet accurate enough
+for us to ensure that a scanned document may be sufficiently accurate for our 
+archival purposes. arXiv is not willing to compromise on this ideal, and we do not
+have the resources to compare individual OCR documents against their original for accuracy. 
+Therefore bitmapped and scanned PDF will be rejected. 

```

Comment:
Therefore bitmapped and scanned PDFs will be rejected.

Reply:
These are all resolved in the most recent version.